deepchecks/checks/distribution/train_test_drift.py

```diff
@@ -27,113 +27,9 @@
-        expected_percents: array of percentages of each value in the expected distribution.
-        actual_percents: array of percentages of each value in the actual distribution.
-        categories_list: list of all categories that the percentages represent.
-
-    """
-    all_categories = list(set(np.unique(dist1)).union(set(dist2)))
-
-    if max_num_categories is not None and len(all_categories) > max_num_categories:
-        dist1_counter = dict(Counter(dist1).most_common(max_num_categories))
-        dist1_counter['Other rare categories'] = len(dist1) - sum(dist1_counter.values())
-        categories_list = list(dist1_counter.keys())
-
-        dist2_counter = Counter(dist2)
-        dist2_counter = {k: dist2_counter[k] for k in categories_list}
-        dist2_counter['Other rare categories'] = len(dist2) - sum(dist2_counter.values())
-
-    else:
-        dist1_counter = Counter(dist1)
-        dist2_counter = Counter(dist2)
-        categories_list = all_categories
-
-    expected_percents = np.array([dist1_counter[k] for k in categories_list]) / len(dist1)
-    actual_percents = np.array([dist2_counter[k] for k in categories_list]) / len(dist2)
-
-    return expected_percents, actual_percents, categories_list
-
-
-def psi(expected_percents: np.ndarray, actual_percents: np.ndarray):
-    """
-    Calculate the PSI (Population Stability Index).
-
-    See https://www.lexjansen.com/wuss/2017/47_Final_Paper_PDF.pdf
-
-    Args:
-        expected_percents: array of percentages of each value in the expected distribution.
-        actual_percents: array of percentages of each value in the actual distribution.
-
-    Returns:
-        psi: The PSI score
-
-    """
-    psi_value = 0
-    for i in range(len(expected_percents)):
-        # In order for the value not to diverge, we cap our min percentage value
-        e_perc = max(expected_percents[i], PSI_MIN_PERCENTAGE)
-        a_perc = max(actual_percents[i], PSI_MIN_PERCENTAGE)
-        value = (e_perc - a_perc) * np.log(e_perc / a_perc)
-        psi_value += value
-
-    return psi_value
-
-
-def earth_movers_distance(dist1: np.ndarray, dist2: np.ndarray):
-    """
-    Calculate the Earth Movers Distance (Wasserstein distance).
-
-    See https://en.wikipedia.org/wiki/Wasserstein_metric
-
-    Function is for numerical data only.
-
-    Args:
-        dist1: array of numberical values.
-        dist2: array of numberical values to compare dist1 to.
-
-    Returns:
-        the Wasserstein distance between the two distributions.
-
-    """
-    unique1 = np.unique(dist1)
-    unique2 = np.unique(dist2)
-
-    sample_space = list(set(unique1).union(set(unique2)))
-
-    val_max = max(sample_space)
-    val_min = min(sample_space)
-
-    if val_max == val_min:
-        return 0
-
-    dist1 = (dist1 - val_min) / (val_max - val_min)
-    dist2 = (dist2 - val_min) / (val_max - val_min)
-
-    return wasserstein_distance(dist1, dist2)
-
-
 class TrainTestDrift(TrainTestBaseCheck):
```

Comment:
Separate thought - shouldn't this be named something like "TrainTestPerFeatureDrift"?
for a clearer indication of what it does

Reply:
Seems a bit clumsy no? 
Maybe TrainTestFeatureDrift is a good compromise
Or drop the TrainTest?